monroe.el

```diff
@@ -390,8 +395,10 @@ will force connection closing, which will as result call '(monroe-sentinel)'."
                                (when (and p (process-live-p p))
                                  (delete-process p))))
         ;; 'monroe-repl-buffer' process is actually 'fake-proc'
-        (proc1 (get-buffer-process monroe-repl-buffer))
-        (proc2 (get-buffer-process "*monroe-connection*")))
+        (proc1 (get-buffer-process
+                (concat monroe-repl-buffer (monroe-locate-running-nrepl-host))))
```

Comment:
It seems like this concat+buffer+locate-running-nrepl-host pattern is repeated many times; I think it would make it easier to change it that were done in a single function, even if it's only one line.

Reply:
I refactored the concat to a function as you suggested with fallback in case .nrepl-port isn't there